sc-tools/sc-server/src/sc-server-impl/sc_server_message_action.hpp

```diff
@@ -64,6 +66,32 @@ class ScServerMessageAction : public ScServerAction
     m_server->Send(hdl, responseText, ScServerMessageType::text);
   }
 
+  void OnHealthCheck(ScServerConnectionHandle const & hdl, ScServerMessage const & msg)
+  {
+    std::string response;
+    try
+    {
+      auto * context = new ScMemoryContext(sc_access_lvl_make_max);
+
+      ScAddr const & tempAddr = context->CreateNode(ScType::NodeConst);
+      context->EraseElement(tempAddr);
+
+      delete context;
+
+      response = "OK";
+      m_server->LogMessage(ScServerLogMessages::app, "I'm alive...");
+    }
+    catch (utils::ScException const & e)
```

Comment:
Healthcheck usually has a timeout. If it expires, a service is considered unhealthy. Do we have an exception which is raised upon some timeout, or we're going to add the timeout in our `healthcheck.py` script?

Reply:
I added the timeout in our `healthcheck.py` script

Reply:
Hmm... I am not sure if it'll work the way I intend it to. For example: we can successfully initiate a connection to sc-server, but memory is in a blocked state and cannot create elements. The timeout will not help, because it is applied to initiating the connection rather than getting the response